src/runtime_src/xdp/profile/database/static_info_database.cpp

```diff
@@ -194,7 +194,7 @@ namespace xdp {
     std::vector<std::string> deviceNames ;
     for (auto device : deviceInfo)
     {
-      deviceNames.push_back((device.second)->deviceName) ;
+      deviceNames.push_back((device.second)->getUniqueDeviceName()) ;
```

Comment:
Since you are here...
```suggestion
      deviceNames.push_back(device.second->getUniqueDeviceName()) ;
```